source/settings.cpp

```diff
@@ -669,6 +671,14 @@ namespace Settings {
   Option ColoredBossKeys = Option::Bool("Colored Boss Keys",  {"Off", "On"}, {coloredBossKeysDesc}, OptionCategory::Cosmetic);
   Option MirrorWorld =     Option::Bool("Mirror World",       {"Off", "On"}, {mirrorWorldDesc},     OptionCategory::Cosmetic);
 
+  static std::vector<std::string> fanfareOptions = {"Off", "Only Fanfares", "Fanfares + Ocarina Music"};
```

Comment:
![image](https://user-images.githubusercontent.com/5352197/127811540-beb29a69-cb48-46c2-8f7a-35982b967190.png)
```suggestion
  static std::vector<std::string> fanfareOptions = {"Off", "Only Fanfares", "Fanfares +\n                         Ocarina Music"};
```
![image](https://user-images.githubusercontent.com/5352197/127811883-9ed5fa7b-b67f-4254-aef9-6cf2db70de3c.png)